### PR TITLE
[gfwx] Add gfwx fuzzers

### PR DESCRIPTION
**Pull request by guidovranken**
As requested in #402.

### CODE REVIEW

File: projects/gfwx/project.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+homepage: "http://www.gfwx.org/"
+primary_contact: "guidovranken@gmail.com"
```
Comment (Dor1s) [inline projects/gfwx/project.yaml]: I think we need to add some maintainer(s) into `auto_ccs`. Otherwise, who will be fixing the bugs? :)

Comment (guidovranken) [reply projects/gfwx/project.yaml]: I just sent an e-mail to the address of the presumed maintainer, that I found [here](http://ict.usc.edu/profile/graham-fyffe/), but it bounced.

I can't find any other contact info or Github profile of him.

He seems to be working for [GoogleVR](https://www.linkedin.com/in/graham-fyffe-84795b1/) (assuming it's the same person) -- is there any chance you can reach out to him internally?

I don't know what else to do.

Comment (Dor1s) [reply projects/gfwx/project.yaml]: Thanks for the reconnaissance, @guidovranken. I've pinged that person internally at Google, will come back once I get the response :)

Comment (Dor1s) [reply projects/gfwx/project.yaml]: Could you please add `fyffe@google.com`?

Comment (guidovranken) [reply projects/gfwx/project.yaml]: Done, thank you for finding the maintainer.

**Pull request merged**